ArcadiaTeamsBot.sln

```diff
@@ -7,11 +7,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ArcadiaTeamsBot", "ArcadiaT
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ArcadiaTeamsBot.CQRS", "ArcadiaTeamsBot.CQRS\ArcadiaTeamsBot.CQRS.csproj", "{A4C32C4B-CEB3-419B-9AF8-BA1E47514717}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ServiceDesk.Abstractions", "ServiceDesk.Abstractions\ServiceDesk.Abstractions.csproj", "{32254F05-44E4-424D-893F-F0CF37AFCB4A}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ArcadiaTeamsBot.ServiceDesk.Abstractions", "ServiceDesk.Abstractions\ArcadiaTeamsBot.ServiceDesk.Abstractions.csproj", "{32254F05-44E4-424D-893F-F0CF37AFCB4A}"

```

Comment:
Project should not only be consistent named, but also lies in consistent folder.

Reply:
Folders have been renamed